src/cap05/SortGradesExercise.java

```diff
@@ -3,38 +3,37 @@
 import java.text.DecimalFormat;
 import java.util.Arrays;
 
+import java.util.Collections;
 import javax.swing.JOptionPane;
 
 public class SortGradesExercise {
 
-	public static void main(String[] args) {
-		float[] grades = new float[5];
+  public static void main(String[] args) {
+    Float[] grades = new Float[5];
```

Comment:
Convertido de um array de float primitivo para array de objetos da classe Float, para poder utilizar o método `reverseOrder` da classe Collections.
Do contrário, seria necessário escrever uma ordenação manual.

Reply:
Exemplo de ordenação manual:
https://www.sanfoundry.com/c-program-sort-array-descending-order/

Reply:
Exemplo:
```suggestion
    float[] grades = new float[5];
    
    for (int i = 0; i < grades.length; i++) {
      for (int j = i + 1; j < grades.length; j++) {
        if (grades[i] < grades[j]) {
          float previousGrade = grades[i];
          grades[i] = grades[j];
          grades[j] = previousGrade;
        }
      }
    }
```
**Explicação**

O primeiro for, com contador `i`, irá fazer o loop pelo array normalmente. Enquanto o for interno, com contador `j`, serve para percorrer um array com sempre uma casa à frente (Enquanto o for externo está no índice `0` o interno estará no índice `1` ), dessa maneira é possível comparar a nota atual com a próxima. 
Se a próxima nota for maior que a nota atual, cria-se uma variável auxiliar para guardar a nota atual (os dados nas posições serão sobrescritos, é **necessário** a variável auxiliar (`previousGrade`), a posição `grades[i]` (que é a posição de trás) irá receber a nota maior que está na posição da frente (`grades[j]`) e a posição `grades[j]` irá receber a nota de trás, que foi guardada na variável auxiliar `previousGrade`.